freqtrade/freqai/tensorboard.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from pathlib import Path
+from typing import Any
+
+import xgboost as xgb
+
+
+logger = logging.getLogger(__name__)
+
+
+class BaseTensorboardLogger:
+    def __init__(self, logdir: str = "tensorboard", id: str = "unique-id"):
+        logger.warning("Tensorboard is not installed, no logs will be written."
+                       "Use ensure torch is installed, or use the torch/RL docker images")
+
+    def log_scaler(self, tag: str, scalar_value: Any, step: int):
+        return
+
+    def close(self):
+        return
+
+
+class BaseTensorBoardCallback(xgb.callback.TrainingCallback):
```

Comment:
Might make sense to import `xgb.callback.TrainingCallback` explicitly (from xgboost.callback import TrainingCallback` ? 
we're now importing xgb directly, but only ever use this one subclass (and a child of that subclass) in this file.